src/commonMain/kotlin/com.adamratzman.spotify/models/SpotifyUris.kt

```diff
@@ -68,12 +58,7 @@ sealed class SpotifyUri(val input: String, type: String) {
         return "SpotifyUri($uri)"
     }
 
-    @Serializer(forClass = SpotifyUri::class)
-    companion object : KSerializer<SpotifyUri> {
-        override val descriptor: SerialDescriptor = StringDescriptor
-        override fun deserialize(decoder: Decoder): SpotifyUri = SpotifyUri(decoder.decodeString())
-        override fun serialize(encoder: Encoder, obj: SpotifyUri) = encoder.encodeString(obj.uri)
-
```

Comment:
Now it's a object again?!
The default parser which will be used now would parse a object like the following:
```
{
    input: string,
    type: string
}
```

But we get a `string`!